common/src/main/java/com/wynntils/mc/render/FontRenderer.java

```diff
@@ -206,11 +223,20 @@ private float calculateRenderHeight(List<TextRenderTask> toRender) {
                     }
                 }
 
-                height += lines * (font.lineHeight + NEWLINE_OFFSET);
+                height += lines * (font.lineHeight) + NEWLINE_OFFSET * (lines - 1);
             }
         }
 
-        return (float) (height - NEWLINE_OFFSET / 2 * McUtils.window().getGuiScale());
+        return (float) (height / 2 * McUtils.window().getGuiScale());
```

Comment:
Not something that needs changing, just me being curious: Why do we divide the height by 2? It seems to me that we do a proper calculation of height. I assumed it was the NEWLINE_OFFSET that were supposed to be divided by 2. But it seems to work, I mean, the alignment look nice and all. I just don't understand why. :-)

Reply:
I am kinda lost in the GUI scaling math stuff, originally it was supposed to be (height - NEWLINE_HEIGHT) / 2 but now newline height is added in a different place.

Reply:
You and me both, brother :-)